docs/HyperSync/HyperRPC/hyperrpc-url-endpoints.md

```diff
@@ -77,3 +77,7 @@ Here is a table of the currently supported networks on HyperRPC and their respec
 
 
 
+
+
+
+
```

Comment:
Arg, this junk is  fixed here: https://github.com/enviodev/docs/pull/404

Reply:
Basically a bug in my script every time I run the script it adds a line. Was a mistake to commit it.